api/types.go

```diff
@@ -537,3 +537,33 @@ func toEc2SecurityGroupResponse(sg *ec2.SecurityGroup) *Ec2SecurityGroupResponse
 		VpcId:         aws.StringValue(sg.VpcId),
 	}
 }
+
+type Ec2VpcResponse struct {
+	Id              string            `json:"id"`
+	CIDRBlock       string            `json:"cidr_block"`
+	DHCPOptionsId   string            `json:"dhcp_options_id"`
+	State           string            `json:"state"`
+	InstanceTenancy string            `json:"instance_tenancy"`
+	IsDefault       bool              `json:"is_default"`
+	Tags            map[string]string `json:"tags"`
```

Comment:
This should be `[]map[string]string` to match the existing format

Reply:
Done.